.github/workflows/e2e.yml

```diff
@@ -0,0 +1,35 @@
+name: End-to-end Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
```

Comment:
A [workflow dispatch event](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) would allow us to run the tests without changes.
```
on: 
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'     
        required: true
        default: 'warning'
      tags:
        description: 'iOS e2e Tests'

Reply:
Added!